auth/principal.go

```diff
@@ -85,6 +85,12 @@ type User interface {
 	// Authenticates the user's password.
 	Authenticate(password string) bool
 
+	// GetSessionUUID returns the UUID that a session to match to be a valid session.
+	GetSessionUUID() []byte
```

Comment:
Is there a reason to store this as []byte instead of string?

Reply:
Bad copy paste, fixed.